src/components/header/scroll-effects.ts

```
@@ -0,0 +1,112 @@
+/**
+ * Scroll Effects Manager Module
+ * Handles header scroll effects and backdrop blur
+ */
+
+export class ScrollEffectsManager {
+  private headers: NodeListOf<HTMLElement> | null = null;
+  private scrollThreshold: number = 20;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)